strings/tips.txt

```diff
@@ -163,6 +163,9 @@ As a Servant, declaring war empowers a huge amount of your tools and constructs,
 As a Servant, converting or sabotaging Science and Genetics can make defending the Ark much easier.
 As a Servant, the Clockwork Arnaments scripture allows you to summon armor and/or a weapon at will. Use it whenever you unlock it!
```

Comment:
arnaments

Reply:
The name is already capitalized and spelled correctly...?

Reply:
>Arnaments

You mean, armaments?

Reply:
Oh, bother!